# Pyxis
Bioinformatics tools to calculate the statistical significance of clusters of genes

Physical clusters of co-regulated, but apparently functionally unrelated genes are present
in many genomes. Despite the important implication that the genomic environment
contributes appreciably to the regulation of gene expression, no simple statistical method
has been described to identify physical clusters of co-regulated genes. Here we report the
development of a model that allows the direct calculation of the significance of such
clusters. We have implemented the derived statistical relation in a software program,
Pyxis, and have analyzed a selection of Saccharomyces cerevisiae gene-expression
microarray data sets. We have identified many gene clusters with constituent genes
exhibiting a regulatory dependence on proteins previously implicated in chromatin
structure. Specifically, we found that Tup1p-dependent gene domains were enriched
close to telomeres, which suggested a new role for Tup1p in telomere silencing. In
addition, we identified Sir2p-, Sir3p- and Sir4p-dependent clusters which suggested the
presence of Sir-mediated heterochromatin in previously unidentified regions of the yeast
genome. We also showed the presence of Sir4p-dependent gene clusters bordering the
HMRa homothallic locus, which suggested leaky termination of the heterochromatin by
the boundary elements. These results demonstrate the utility of Pyxis in identifying
possible higher-order genomic features that may contribute to gene regulation in
extended domains.

See https://academic.oup.com/nar/article/32/5/1798/2380664
